analyzer/src/funTest/assets/projects/external/multi-kotlin-project-expected-output-cli.yml

```diff
@@ -15,222 +15,95 @@ project:
     revision: "<REPLACE_REVISION>"
     path: "analyzer/src/funTest/assets/projects/external/multi-kotlin-project/cli"
   homepage_url: ""
-  scopes:
-  - name: "annotationProcessor"
-    dependencies: []
-  - name: "apiDependenciesMetadata"
-    dependencies:
-    - id: "Gradle:org.gradle.kotlin.dsl.samples.multiproject:core:1.0"
-      linkage: "PROJECT_DYNAMIC"
-      dependencies:
-      - id: "Maven:org.jetbrains.kotlin:kotlin-stdlib:1.3.11"
+  dependency_graph:
+    packages:
+    - "Gradle:org.gradle.kotlin.dsl.samples.multiproject:core:1.0"
+    - "Maven:org.jetbrains.kotlin:kotlin-stdlib:1.3.11"
+    - "Maven:org.jetbrains.kotlin:kotlin-stdlib-common:1.3.11"
+    - "Maven:org.jetbrains:annotations:13.0"
+    - "Maven:org.jetbrains.kotlin:kotlin-compiler-embeddable:1.3.11"
+    - "Maven:org.jetbrains.kotlin:kotlin-script-runtime:1.3.11"
+    - "Maven:org.jetbrains.kotlin:kotlin-reflect:1.3.11"
+    - "Maven:org.jetbrains.kotlin:kotlin-scripting-compiler-embeddable:1.3.11"
+    entry_nodes:
+    - dependencies:
+      - index: 1
```

Comment:
Consider "package_index", or maybe "pkg_index".

Reply:
Renamed to "pkg_index"